### ISSUE TITLE
No account submodule within az billing

### ISSUE THREAD

**Issue by zachgatesak** [Azure/azure-cli#16050]
I've tested from an updated CLI as well as from the cloud shell. Any running of `az billing account ....` yields:
```bash
zach@Azure:~$ az billing account
CommandNotFoundError: 'account' is misspelled or not recognized by the system.
Try this: 'az billing enrollment-account list'
Still stuck? Run 'az billing --help' to view all commands or go to 'https://docs.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest' to learn more
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 68248b96-9021-0508-a9c6-7abf0ec194d6
* Version Independent ID: ad05d26e-9365-f6e1-6b0d-1556f909fe9e
* Content: [az billing account](https://docs.microsoft.com/en-us/cli/azure/billing/account?view=azure-cli-latest#az_billing_account_list)
* Content Source: [latest/docs-ref-autogen/billing/account.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/billing/account.yml)
* Service: **cost-management-billing**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by zachgatesak** [comment]
```Bash
zach@Azure:~$ az version
{
  "azure-cli": "2.14.0",
  "azure-cli-core": "2.14.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {
    "ai-examples": "0.2.4"
  }
}
```

**Comment by yonzhan** [comment]
Billing

**Comment by haroldrandom** [comment]
hi, @zachgatesak the upgraded billing commands are shipped with Azure CLI 2.15.0 and the Cloud Shell you are using with Azure CLI is still 2.14.0. 
I saw Cloud Shell has onboarded Azure CLI 2.15.0 already, you could try it.
